.github/workflows/labeler.yml

```
@@ -77,7 +78,8 @@
         uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)